mod/dagmodifier.go

```diff
@@ -48,24 +51,36 @@ type DagModifier struct {
 	Prefix         cid.Prefix
 	RawLeaves      bool
 	BalancedFormat bool
-	MetadataDag    bool
```

Comment:
Just for my information, is this removed because it's an old bool that's no longer used?

Reply:
Yes. it is correct. It is not used anymore.